Assets/Scripts/Databases/BountyRewardIconDatabase.cs

```diff
@@ -0,0 +1,15 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+[CreateAssetMenu(fileName = "New Bounty Reward Icon Database", menuName = "Bounty Reward Icon Database")]
+public class BountyRewardIconDatabase : ScriptableObject
+{
+    [System.Serializable]
+    public class BountyNameIconPair
+    {
+        [field: SerializeField] public string BountyName { get; set; }
```

Comment:
I can do something like this (ideally provided with a doc on what exactly it would look like), but at the end of the day, I think some bounty related stuff will be hard-coded since we can't map a scriptable object to behaviour.

Reply:
Yeah, I've been seriously brainstorming recently about how to reduce as much editor hard coding as possible. I'll probably tinker around with some stuff and see what sticks.